src/PowerShellEditorServices.Engine/Services/Symbols/SymbolsService.cs

```diff
@@ -227,5 +229,38 @@ public List<SymbolReference> FindSymbolsInFile(ScriptFile scriptFile)
 
             return symbolReference;
         }
+
+        /// <summary>
+        /// Finds the details of the symbol at the given script file location.
+        /// </summary>
+        /// <param name="scriptFile">The ScriptFile in which the symbol can be located.</param>
+        /// <param name="lineNumber">The line number at which the symbol can be located.</param>
+        /// <param name="columnNumber">The column number at which the symbol can be located.</param>
+        /// <returns></returns>
+        public async Task<SymbolDetails> FindSymbolDetailsAtLocationAsync(
+            ScriptFile scriptFile,
+            int lineNumber,
+            int columnNumber,
+            PowerShellContextService powerShellContext)
+        {
+            SymbolReference symbolReference =
+                AstOperations.FindSymbolAtPosition(
+                    scriptFile.ScriptAst,
+                    lineNumber,
+                    columnNumber);
+
+            if (symbolReference == null)
+            {
+                // TODO #21: Return Result<T>
```

Comment:
What does this TODO mean? Is it obsolete now?

Reply:
https://github.com/PowerShell/PowerShellEditorServices/issues/21 yep

Reply:
deleted